src/QsCompiler/CompilationManager/EditorSupport/CodeActions.cs

```diff
@@ -189,6 +230,50 @@ private static IEnumerable<TextEdit> OpenDirectiveSuggestions(this FileContentMa
                 .Select(edit => (edit.NewText.Trim().Trim(';'), file.GetWorkspaceEdit(edit)));
         }
 
+        /// <summary>
+        /// Returns a sequence of replacement Type suggestions for how errors for unknown types and callable in the given diagnostics can be fixed,
+        /// given the file for which those diagnostics were generated and the corresponding compilation.
+        /// Returns an empty enumerable if any of the given arguments is null.
+        /// </summary>
+        internal static IEnumerable<(string, WorkspaceEdit)> CapitalizationSuggestionsForUnknownIdentifiers
```

Comment:
For me, it wasn't suggesting anything when the type was part of an opened namespace (rather than defined in the same file). So for example, this gave me no suggestions for `Complex`:

```
namespace Project {
    open Microsoft.Quantum.Math;

    @EntryPoint()
    operation HelloQ() : Unit {
        let x = complex(2.0, 5.0);
    }
}
```

I'm not sure what is causing that, though.

Reply:
Probably caused by point 1 here: https://github.com/microsoft/qsharp-compiler/pull/266#discussion_r513816570